sys/random/Kconfig

```diff
@@ -16,7 +16,7 @@ if MODULE_RANDOM
 choice RANDOM_IMPLEMENTATION
     bool "PRNG Implementation"
     depends on TEST_KCONFIG
-    default MODULE_PRNG_HWRNG if HAS_PERIPH_HWRNG
+    default MODULE_PRNG_HWRNG if HAS_PERIPH_HWRNG && !MODULE_NETDEV_DEFAULT
```

Comment:
why?

Reply:
To be honest, I don't remember but there was a problem when module `netdev_default` is used.